test/parallel/test-eventtarget.js

```diff
@@ -0,0 +1,229 @@
+'use strict';
```

Comment:
Let me know if you want me to start pushing tests from the WPT and/or browsers here, at another branch or elsewhere.

Reply:
@benjamingr if you'd like to do this, PTAL at https://github.com/nodejs/node/tree/master/test/wpt#how-to-add-tests-for-a-new-module

Reply:
@targos that's super useful thanks, I didn't know we already had a clear process documented in line for this.

Reply:
Let's old off, but it would be good to know which WPT tests for `EventTarget` and `Event` we don't pass.